charts/posthog/templates/recordings-blob-ingestion-deployment.yaml

```diff
@@ -0,0 +1,38 @@
+{{ if .Values.recordingsBlobIngestion.enabled }}
```

Comment:
Can you add a unittest for this deployment? Just basic tests. I guess specific ones for S3 details would be sensible.

Reply:
Yeah pretty much 😬  but perhaps tailor to the specific env vars you abs need.

It will at least ensure that enable: true etc works as expected.

Reply:
I added a test but not for the envs, largely because they aren't different to the existing object storage ones.